packages/lambda-tiler/src/index.ts

```diff
@@ -20,6 +21,10 @@ export const handler = lf.http(LogConfig.get());
 handler.router.hook('request', (req) => {
   req.set('name', 'LambdaTiler');
 
+  // Set the asset location for asset provider
+  const assetLocation = Env.get(Env.AssetLocation);
```

Comment:
this is where the `req.query.get('config')` logic should live.

Reply:
Are we just replace the asset location at the moment, or we can replace the whole configProvider?

Reply:
at the moment we can just have a single `Asset` provider.

longer term we will make `req.assets` and `req.config` so the request has it all and we no longer be using global statics.